File-Formats/PDF/Working-with-Tagged-PDF.md

```diff
@@ -2906,3 +2906,184 @@ Follow the below steps to validate the tagged PDF information in Adobe Acrobat:
 The following screenshot shows the Accessibility checker dialog box.
 
 ![TaggedPDF](TaggedPDF_images/TaggedPDF_img0.png) 
+
+## Get the accessibility tags
+
+Added support for extracting accessibility tags from PDF documents.
```

Comment:
You can extract the existing tag details by using the **StructureElement** property available on the **PdfLoadedDocument** class. The following code explains this.

Reply:
Hi @chinnumuniyappan,

The suggested changes has been updated.

Regards,
Anantha Gokula Raman J